snapcraft/internal/common.py

```diff
@@ -42,7 +42,9 @@
 _keyringsdir = _DEFAULT_KEYRINGSDIR
 _DEFAULT_LEGACY_SNAPCRAFT_DIR = os.path.join(sys.prefix, "legacy_snapcraft")
 _legacy_snapcraft_dir = _DEFAULT_LEGACY_SNAPCRAFT_DIR
+
 _DOCKERENV_FILE = "/.dockerenv"
+_PODMAN_FLAG = "/run/.containerenv"
```

Comment:
Why did you call this FLAG and not FILE?

Reply:
Just to remind that it is just an empty file. Fixed.

Reply:
Renamed to `is_process_container` to be consistent with `is_snap` above and to distinguish between LXD full OS containers.